src/cache/clear_site_data/mod.rs

```diff
@@ -0,0 +1,308 @@
+//! Clear browsing data (cookies, storage, cache) associated with the
+//! requesting website
+
+use crate::headers::{HeaderName, HeaderValue, Headers, ToHeaderValues, CLEAR_SITE_DATA};
+
+use std::fmt::{self, Debug, Write};
+use std::iter::Iterator;
+use std::option;
+use std::slice;
+use std::str::FromStr;
+
+mod directive;
+
+pub use directive::ClearDirective;
+
+/// Clear browsing data (cookies, storage, cache) associated with the
+/// requesting website.
+///
+/// [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data)
+///
+/// # Specifications
+///
+/// - [RFC 7232, section 3.1: If-Match](https://tools.ietf.org/html/rfc7232#section-3.1)
```

Comment:
This is incorrect but I also cant find an rfc for this? Maybe I don't know how to search for these rfcs...

Reply:
https://w3c.github.io/webappsec-clear-site-data/ — i tend to go to mdn and then check their reference section